content/applications/services/field_service/creating_tasks.rst

```diff
@@ -0,0 +1,80 @@
+============================
+Creating Field Service Tasks
+============================
+
+Creating tasks in **Field Service** from a sales order or a helpdesk ticket is faster and easier
+since these already contain a lot of information that will be imported automatically into your field
+service task, but it is also possible to create one manually from scratch.
+
+Manually
+========
+
+ #. Go to :guilabel:`Field Service`
+ #. Click on :guilabel:`New`
+ #. Fill in the :guilabel:`Task Title` and :guilabel:`Customer` (all other fields are optional),
+    then click :guilabel:`Save`
+
+Task configuration
+------------------
+
+Click the task to open it. The task form includes the following fields that you can fill in:
+
+ - :guilabel:`Task Title`: title of the task.
+ - :icon:`fa-star-o` (:guilabel:`Star`): click the :icon:`fa-star-o` (:guilabel:`star`) icon to mark
+   the task as high priority. The icon will turn yellow. Click it again to remove the high priority.
+ - :guilabel:`Project`: the project that this task belongs to.
+ - :guilabel:`Worksheet Template`: the worksheet template you want to use for this task.
+ - :guilabel:`Assignees`: the person(s) in charge of handling the work on this task.
+ - :guilabel:`Tags`: custom labels allowing to categorize and filter your tasks.
+ - :guilabel:`Customer`: the person or company that will be billed for this task. This field only
+   appears in tasks that belong to billable projects.
+ - :guilabel:`Contact Number`: your customer's phone number
+ - :guilabel:`Sales Order Item`: this can be either the sales order that was used to create this
+   task, or a sales order that was linked to this task manually. This field only appears in tasks
+   linked to billable projects.
+ - :guilabel:`Under Warranty` check box: if ticked, the time and materials used for this task will
+   not be billed to the customer. However, the inventory of consumed materials will still be
+   updated.
+ - :guilabel:`Planned Date`: the date you want to schedule for this field service.
+ - :guilabel:`Allocated Time`: the amount of time that the work on this task is expected to last,
+   tracked by timesheets.
+
+From a sales order
+==================
+
+Allowing your sales team to create field service tasks offers a seamless experience for your
+customers. They can receive a quotation they first have to approve before the work even starts. To
+do this, you first need to create a specific product that you can sell as part of a field service.
```

Comment:
- This is a little bit too marketing-y I think: see this for a better example https://www.odoo.com/documentation/master/applications/services/project/tasks/task_creation.html#creating-tasks-from-an-email-alias 
- And since the content is a duplicate from this https://www.odoo.com/documentation/master/applications/sales/sales/invoicing/time_materials.html#service-product-configuration (and we try to avoid that) but we still want to mention somewhere here that you can create FS tasks from a sales order, we could rephrase this whole section to only keep: 1) the fact we can create FS tasks from a sales order 2) that you need a "service" product (add a ref to this as well https://www.odoo.com/documentation/master/applications/sales/sales/invoicing/time_materials.html#service-product-configuration and keep the screenshot! I think it's useful) and 3) you could keep lines 63-65 to mention the process in general.
- Let's discuss it, it's getting complicated xD